README.md

```diff
@@ -60,7 +65,7 @@ export type StreamProps = {
    */
   controls?: boolean;
   /**
-   * Returns the current playback time in seconds. Setting this value seeks the video to a new time.
+   * Setting this value seeks the video to a new time.
```

Comment:
It may be worth mentioning that if a user wants to seek multiple times to the same point, they need to use streamRef or they'll run into https://github.com/cloudflare/stream-react/issues/14